ProcessesApi/V1/Gateways/ProcessesGateway.cs

```diff
@@ -42,5 +60,20 @@ public async Task<Process> CreateNewProcess(CreateProcessQuery query, string pro
             await _dynamoDbContext.SaveAsync(processDbEntity).ConfigureAwait(false);
             return processDbEntity.ToDomain();
         }
+
+        [LogCall]
+        public async Task<Process> UpdateProcess(UpdateProcessQueryObject requestObject, UpdateProcessQuery query, int? ifMatch)
```

Comment:
This is a bit picky, but would it be possible to add this method and the `CreateUpdatedProcess()` next to each other (rather than having the `getProcess()` and `CreateProcess()` in between)